flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/table/FileSystemTableFactory.java

```diff
@@ -79,9 +86,27 @@ public DynamicTableSource createDynamicTableSource(Context context) {
     @Override
     public DynamicTableSink createDynamicTableSink(Context context) {
         FactoryUtil.TableFactoryHelper helper = FactoryUtil.createTableFactoryHelper(this, context);
+        helper.forwardOptions(
+                FileSystemConnectorOptions.PATH,
+                FileSystemConnectorOptions.PARTITION_DEFAULT_NAME,
+                FileSystemConnectorOptions.PARTITION_TIME_EXTRACTOR_KIND,
+                FileSystemConnectorOptions.PARTITION_TIME_EXTRACTOR_CLASS,
+                FileSystemConnectorOptions.PARTITION_TIME_EXTRACTOR_TIMESTAMP_PATTERN,
+                FileSystemConnectorOptions.SINK_ROLLING_POLICY_FILE_SIZE,
+                FileSystemConnectorOptions.SINK_ROLLING_POLICY_ROLLOVER_INTERVAL,
+                FileSystemConnectorOptions.SINK_ROLLING_POLICY_CHECK_INTERVAL,
+                FileSystemConnectorOptions.SINK_SHUFFLE_BY_PARTITION,
```

Comment:
this sounds dangerous. let's exclude everything except for PATH in the first version? Otherwise we should ask someone from the SDK team.

Reply:
Shuffle by partition is a mistake, will revert it. But the rolling policy sounds safe as it's just tuning the object `TableRollingPolicy`